contrib/xz-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,246 @@
+#############################################################################
```

Comment:
But `xz` library already contains CMakeLists is it unusable?

Reply:
reused `xz` CMakeLists